src/module.js

```diff
@@ -213,24 +208,24 @@ const PARTICIPANT_PROPS = {
   },
   setSubscribedTracks: {
     validate: (subs, callObject, participant) => {
-      if (!callObject._callObjectMode) {
-        return false;
-      }
       if (callObject._preloadCache.subscribeToTracksAutomatically) {
         return false;
       }
       if (participant.local) {
         return false;
       }
+      if ([true, false, 'avatar'].includes(subs)) {
+        return true;
+      }
       for (const s in subs) {
         if (!(s === 'audio' || s ===  'video' || s === 'screenVideo')) {
           return false;
         }
       }
       return true;
     },
-    help: 'setSubscribedTracks can only be used in call object mode, cannot be used on the local participant, cannot be used when setSubscribeToTracksAutomatically is enabled, and should be of the form: ' +
-      'true | false | { [audio: true|false], [video: true|false], [screenVideo: true|false] }'
+    help: 'setSubscribedTracks cannot be used on the local participant, cannot be used when setSubscribeToTracksAutomatically is enabled, and should be of the form: ' +
```

Comment:
did you also want to go ahead and `screenAudio`, since you've been working on adding support for that as well?

Reply:
hm, nvm, actually it looks like it's not yet an individually subscribable track, right? the only way to subscribe to it is to subscribe to all tracks?